src/app/tests/TestReadInteraction.cpp

```diff
@@ -185,8 +185,8 @@ class MockInteractionModelApp : public chip::app::ReadClient::Callback, public c
 
 namespace chip {
 namespace app {
-CHIP_ERROR ReadSingleClusterData(const Access::SubjectDescriptor & aSubjectDescriptor, const ConcreteReadAttributePath & aPath,
-                                 AttributeReportIBs::Builder & aAttributeReports,
+CHIP_ERROR ReadSingleClusterData(const Access::SubjectDescriptor & aSubjectDescriptor, bool aIsFabricFiltered,
+                                 const ConcreteReadAttributePath & aPath, AttributeReportIBs::Builder & aAttributeReports,
```

Comment:
What about YAML tests?

Also, can we add a test in the controller data model tests to actually create two fabrics, then read out the data from one fabric vs the other?

Reply:
Resolve as TODO #13444